app/controllers/v0/sign_in_controller.rb

```diff
@@ -161,7 +163,8 @@ def revoke
     end
 
     def revoke_all_sessions
-      SignIn::RevokeSessionsForUser.new(user_account: @current_user.user_account).perform
+      session = SignIn::OAuthSession.find_by(handle: @access_token.session_handle)
```

Comment:
I tested this and ran `revoke_all_sessions` twice, so the second time the access token was still valid but `session` didn't exist. This makes line 167 crash hard. So I suppose we need to specifically raise a `SessionNotFoundError` if this is `nil`, that way we'll more elegantly fail

Reply:
but that was the only thing I found from doing multiple `revoke_all_sessions` and `logout` tests